SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/WorkerRegion.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "LoadBalancing/WorkerRegion.h"
+
+#include "Materials/MaterialInstanceDynamic.h"
+#include "UObject/ConstructorHelpers.h"
+#include "UObject/UObjectGlobals.h"
+
+DEFINE_LOG_CATEGORY(LogWorkerRegion);
+
+const float AWorkerRegion::DEFAULT_WORKER_REGION_HEIGHT = 10;
+const float AWorkerRegion::DEFAULT_WORKER_REGION_OPACITY = 0.7;
+
+AWorkerRegion::AWorkerRegion(const FObjectInitializer& ObjectInitializer)
+	: Super(ObjectInitializer)
+{
+	Mesh = ObjectInitializer.CreateDefaultSubobject<UStaticMeshComponent>(this, TEXT("WorkerRegionPlane"));
```

Comment:
could consider moving these hardcode strings to settings set in BP_SpatialDebugger, the same way the image icon references are set

Reply:
This is just the actor name?